diff --git a/tools/dbus-monitor.c b/tools/dbus-monitor.c
index 9f7da5b..c66186f 100644
--- a/tools/dbus-monitor.c
+++ b/tools/dbus-monitor.c
@@ -33,8 +33,6 @@
 
 #include <time.h>
 
-#include <signal.h>
-
 #include "dbus-print-message.h"
 
 #ifdef DBUS_WIN
@@ -195,14 +193,6 @@ usage (char *name, int ecode)
   exit (ecode);
 }
 
-dbus_bool_t sigint_received = FALSE;
-
-static void
-sigint_handler (int signum)
-{
-  sigint_received = TRUE;
-}
-
 int
 main (int argc, char *argv[])
 {
@@ -294,11 +284,7 @@ main (int argc, char *argv[])
     fprintf (stderr, "Couldn't add filter!\n");
     exit (1);
   }
-
-  /* we handle SIGINT so exit() is reached and flushes stdout */
-  signal (SIGINT, sigint_handler);
-  while (dbus_connection_read_write_dispatch(connection, -1)
-          && !sigint_received)
+  while (dbus_connection_read_write_dispatch(connection, -1))
     ;
   exit (0);
  lose:
