diff --git a/dbus/dbus-sysdeps-unix.c b/dbus/dbus-sysdeps-unix.c
index 2ae7909..4730c8e 100644
--- a/dbus/dbus-sysdeps-unix.c
+++ b/dbus/dbus-sysdeps-unix.c
@@ -1247,7 +1247,7 @@ _dbus_read_credentials_socket  (int              client_fd,
 #elif defined(HAVE_CMSGCRED)
     struct cmsgcred *cred;
 
-    cred = (struct cmsgcred *) CMSG_DATA (&cmsg);
+    cred = (struct cmsgcred *) CMSG_DATA (&cmsg.hdr);
     pid_read = cred->cmcred_pid;
     uid_read = cred->cmcred_euid;
 #elif defined(LOCAL_CREDS)
