diff -Naur dbus-0.94.vanilla/Makefile.in dbus-0.94/Makefile.in
--- dbus-0.94.vanilla/Makefile.in	2006-10-24 10:03:52.000000000 +0200
+++ dbus-0.94/Makefile.in	2006-10-24 10:11:02.000000000 +0200
@@ -176,6 +176,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -222,17 +223,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -249,23 +248,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
diff -Naur dbus-0.94.vanilla/bus/Makefile.in dbus-0.94/bus/Makefile.in
--- dbus-0.94.vanilla/bus/Makefile.in	2006-10-24 10:03:52.000000000 +0200
+++ dbus-0.94/bus/Makefile.in	2006-10-24 10:10:59.000000000 +0200
@@ -205,6 +205,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -251,17 +252,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -278,23 +277,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
diff -Naur dbus-0.94.vanilla/dbus/Makefile.in dbus-0.94/dbus/Makefile.in
--- dbus-0.94.vanilla/dbus/Makefile.in	2006-10-24 10:03:52.000000000 +0200
+++ dbus-0.94/dbus/Makefile.in	2006-10-24 10:11:00.000000000 +0200
@@ -197,6 +197,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -243,17 +244,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -270,23 +269,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
@@ -296,7 +302,7 @@
 target_os = @target_os@
 target_vendor = @target_vendor@
 configdir = $(sysconfdir)/dbus-1
-INCLUDES = -I$(top_builddir) -I$(top_srcdir) $(DBUS_CLIENT_CFLAGS) -DDBUS_COMPILATION -DDBUS_MACHINE_UUID_FILE=\""$(configdir)/machine-id"\"
+INCLUDES = -I$(top_builddir) -I$(top_srcdir) $(DBUS_CLIENT_CFLAGS) -DDBUS_COMPILATION -DDBUS_MACHINE_UUID_FILE=\""$(localstatedir)/lib/dbus/machine-id"\"
 dbusincludedir = $(includedir)/dbus-1.0/dbus
 dbusarchincludedir = $(libdir)/dbus-1.0/include/dbus
 lib_LTLIBRARIES = libdbus-1.la
diff -Naur dbus-0.94.vanilla/doc/Makefile.in dbus-0.94/doc/Makefile.in
--- dbus-0.94.vanilla/doc/Makefile.in	2006-10-24 10:03:52.000000000 +0200
+++ dbus-0.94/doc/Makefile.in	2006-10-24 10:11:00.000000000 +0200
@@ -130,6 +130,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -176,17 +177,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -203,23 +202,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
diff -Naur dbus-0.94.vanilla/test/Makefile.in dbus-0.94/test/Makefile.in
--- dbus-0.94.vanilla/test/Makefile.in	2006-10-24 10:03:51.000000000 +0200
+++ dbus-0.94/test/Makefile.in	2006-10-24 10:11:01.000000000 +0200
@@ -198,6 +198,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -244,17 +245,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -271,23 +270,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
diff -Naur dbus-0.94.vanilla/test/name-test/Makefile.in dbus-0.94/test/name-test/Makefile.in
--- dbus-0.94.vanilla/test/name-test/Makefile.in	2006-10-24 10:03:51.000000000 +0200
+++ dbus-0.94/test/name-test/Makefile.in	2006-10-24 10:11:01.000000000 +0200
@@ -166,6 +166,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -212,17 +213,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -239,23 +238,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
diff -Naur dbus-0.94.vanilla/tools/Makefile.in dbus-0.94/tools/Makefile.in
--- dbus-0.94.vanilla/tools/Makefile.in	2006-10-24 10:03:52.000000000 +0200
+++ dbus-0.94/tools/Makefile.in	2006-10-24 10:11:01.000000000 +0200
@@ -175,6 +175,7 @@
 F77 = @F77@
 FFLAGS = @FFLAGS@
 GETTEXT_PACKAGE = @GETTEXT_PACKAGE@
+GREP = @GREP@
 HAVE_CONSOLE_OWNER_FILE_FALSE = @HAVE_CONSOLE_OWNER_FILE_FALSE@
 HAVE_CONSOLE_OWNER_FILE_TRUE = @HAVE_CONSOLE_OWNER_FILE_TRUE@
 HAVE_SELINUX_FALSE = @HAVE_SELINUX_FALSE@
@@ -221,17 +222,15 @@
 TEST_SLEEP_FOREVER_BINARY = @TEST_SLEEP_FOREVER_BINARY@
 TEST_SOCKET_DIR = @TEST_SOCKET_DIR@
 VERSION = @VERSION@
+XMKMF = @XMKMF@
 XMLTO = @XMLTO@
 X_CFLAGS = @X_CFLAGS@
 X_EXTRA_LIBS = @X_EXTRA_LIBS@
 X_LIBS = @X_LIBS@
 X_PRE_LIBS = @X_PRE_LIBS@
-ac_ct_AR = @ac_ct_AR@
 ac_ct_CC = @ac_ct_CC@
 ac_ct_CXX = @ac_ct_CXX@
 ac_ct_F77 = @ac_ct_F77@
-ac_ct_RANLIB = @ac_ct_RANLIB@
-ac_ct_STRIP = @ac_ct_STRIP@
 am__fastdepCC_FALSE = @am__fastdepCC_FALSE@
 am__fastdepCC_TRUE = @am__fastdepCC_TRUE@
 am__fastdepCXX_FALSE = @am__fastdepCXX_FALSE@
@@ -248,23 +247,30 @@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
+datarootdir = @datarootdir@
+docdir = @docdir@
+dvidir = @dvidir@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
 host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
+htmldir = @htmldir@
 includedir = @includedir@
 infodir = @infodir@
 install_sh = @install_sh@
 libdir = @libdir@
 libexecdir = @libexecdir@
+localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
 oldincludedir = @oldincludedir@
+pdfdir = @pdfdir@
 prefix = @prefix@
 program_transform_name = @program_transform_name@
+psdir = @psdir@
 sbindir = @sbindir@
 sharedstatedir = @sharedstatedir@
 sysconfdir = @sysconfdir@
@@ -274,7 +280,7 @@
 target_os = @target_os@
 target_vendor = @target_vendor@
 configdir = $(sysconfdir)/dbus-1
-INCLUDES = -I$(top_srcdir) $(DBUS_CLIENT_CFLAGS) $(DBUS_X_CFLAGS) -DDBUS_LOCALEDIR=\"@EXPANDED_DATADIR@/locale\" -DDBUS_COMPILATION -DDBUS_DAEMONDIR=\"@DBUS_DAEMONDIR@\" -DDBUS_MACHINE_UUID_FILE=\""$(configdir)/machine-id"\"
+INCLUDES = -I$(top_srcdir) $(DBUS_CLIENT_CFLAGS) $(DBUS_X_CFLAGS) -DDBUS_LOCALEDIR=\"@EXPANDED_DATADIR@/locale\" -DDBUS_COMPILATION -DDBUS_DAEMONDIR=\"@DBUS_DAEMONDIR@\" -DDBUS_MACHINE_UUID_FILE=\""$(localstatedir)/lib/dbus/machine-id"\"
 dbus_send_SOURCES = \
 	dbus-print-message.c			\
 	dbus-print-message.h			\
