Index: dbus/dbus-transport-unix.c
===================================================================
RCS file: /cvs/dbus/dbus/dbus/dbus-transport-unix.c,v
retrieving revision 1.48
retrieving revision 1.49
diff -u -r1.48 -r1.49
--- dbus/dbus-transport-unix.c	25 Oct 2005 15:57:13 -0000	1.48
+++ dbus/dbus-transport-unix.c	30 May 2006 15:34:10 -0000	1.49
@@ -1272,16 +1272,18 @@
       return NULL;
     }
   
-  if (!_dbus_string_append (&address, "tcp:host=") ||
-      !_dbus_string_append (&address, host) ||
-      !_dbus_string_append (&address, ",port=") ||
+  if (!_dbus_string_append (&address, "tcp:"))
+    goto error;
+
+  if (host != NULL && 
+       (!_dbus_string_append (&address, "host=") ||
+        !_dbus_string_append (&address, host)))
+    goto error;
+
+  if (!_dbus_string_append (&address, ",port=") ||
       !_dbus_string_append_int (&address, port))
-    {
-      _dbus_string_free (&address);
-      dbus_set_error (error, DBUS_ERROR_NO_MEMORY, NULL);
-      return NULL;
-    }
-  
+    goto error;
+
   fd = _dbus_connect_tcp_socket (host, port, error);
   if (fd < 0)
     {
@@ -1307,6 +1309,11 @@
   _dbus_string_free (&address);
   
   return transport;
+
+error:
+  _dbus_string_free (&address);
+  dbus_set_error (error, DBUS_ERROR_NO_MEMORY, NULL);
+  return NULL;
 }
 
 /** @} */
