--- configure.in.old	2006-04-30 20:23:40.706665752 +0200
+++ configure.in	2006-04-30 20:24:49.347230800 +0200
@@ -297,30 +297,21 @@
 if test x$enable_mono_docs = xyes; then
     if test x$have_monodoc = xno ; then
 	AC_MSG_ERROR([Building Mono docs explicitly required, but monodoc not found])
-    else
-        AC_PATH_PROG(MONODOC, monodoc, no)
-        if test x$MONODOC = xno; then
-           AC_MSG_ERROR([Building Mono docs explicitly required, but monodoc not found])	
-	fi
     fi
 fi
 
 if test x$enable_mono_docs = xauto ; then
     if test x$have_monodoc = xno ; then
         enable_mono_docs=no
-	MONODOC=
     else
-        AC_PATH_PROG(MONODOC, monodoc, no)
         if test x$MONODOC = xno; then
            enable_mono_docs=no
-           MONODOC=
 	else
 	   enable_mono_docs=yes
         fi
     fi
 fi
 
-AC_SUBST(MONODOC)
 
 AM_CONDITIONAL(ENABLE_MONODOC, test "x$enable_mono_docs" = "xyes")
 
--- mono/doc/Makefile.am.old	2006-04-30 21:07:00.066502992 +0200
+++ mono/doc/Makefile.am	2006-04-30 21:07:27.942265232 +0200
@@ -1,17 +1,17 @@
 if ENABLE_MONODOC
 TARGETS = dbus-sharp-docs.zip dbus-sharp-docs.tree dbus-sharp-docs.source
-DOCDIR = `$(MONODOC) --get-sourcesdir`
+DOCDIR = `pkg-config monodoc --variable=sourcesdir`
 
 install-data-local:
-	install -d -m 755 $(DESTDIR)/`monodoc --get-sourcesdir`
-	install -m 644 $(srcdir)/dbus-sharp-docs.source $(DESTDIR)/`monodoc --get-sourcesdir`
-	install -m 644 dbus-sharp-docs.tree $(DESTDIR)/`monodoc --get-sourcesdir`
-	install -m 644 dbus-sharp-docs.zip $(DESTDIR)/`monodoc --get-sourcesdir`
+	install -d -m 755 $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`
+	install -m 644 $(srcdir)/dbus-sharp-docs.source $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`
+	install -m 644 dbus-sharp-docs.tree $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`
+	install -m 644 dbus-sharp-docs.zip $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`
 
 uninstall-local:
-	rm -f $(DESTDIR)/`monodoc --get-sourcesdir`/dbus-sharp-docs.source
-	rm -f $(DESTDIR)/`monodoc --get-sourcesdir`/dbus-sharp-docs.tree
-	rm -f $(DESTDIR)/`monodoc --get-sourcesdir`/dbus-sharp-docs.zip
+	rm -f $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`/dbus-sharp-docs.source
+	rm -f $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`/dbus-sharp-docs.tree
+	rm -f $(DESTDIR)/`pkg-config monodoc --variable=sourcesdir`/dbus-sharp-docs.zip
 else
 TARGETS = 
 DOCDIR = $(datadir)
@@ -19,8 +19,8 @@
 
 noinst_DATA = $(TARGETS)
 
-ASSEMBLER = $(MONODOC) --assemble
-UPDATER = $(MONODOC) --update
+ASSEMBLER = mdassembler
+UPDATER = mdassembler
 
 dbus-sharp-docs.zip: dbus-sharp-docs.tree
 
@@ -28,7 +28,7 @@
 	$(ASSEMBLER) --ecma $(srcdir)/en -o dbus-sharp-docs
 
 update:
-		$(UPDATER) ../dbus-sharp.dll -o ./en -f
+	$(ASSEMBLER) --ecma $(srcdir)/en -o dbus-sharp-docs
 
 CLEANFILES = dbus-sharp-docs.zip dbus-sharp-docs.tree
 
