Index: bus/driver.c
===================================================================
RCS file: /cvs/dbus/dbus/bus/driver.c,v
retrieving revision 1.73
diff -u -r1.73 driver.c
--- bus/driver.c	3 Oct 2005 19:55:56 -0000	1.73
+++ bus/driver.c	30 Oct 2005 22:21:07 -0000
@@ -1094,23 +1094,35 @@
 				 DBusError      *error)
 {
   BusContext *context;
-  dbus_bool_t retval;
+  DBusMessage *reply;
 
   _DBUS_ASSERT_ERROR_IS_CLEAR (error);
 
-  retval = FALSE;
-
   context = bus_connection_get_context (connection);
   if (!bus_context_reload_config (context, error))
     {
-      _DBUS_ASSERT_ERROR_IS_SET (error);
-      goto out;
+      goto failed;
     }
 
-  retval = TRUE;
-  
+  reply = dbus_message_new_method_return (message);
+  if (reply == NULL)
+    goto oom;
+
+  if (! bus_transaction_send_from_driver (transaction, connection, reply))
+    goto oom;
+
+  dbus_message_unref (reply);
+  return TRUE;
+
+ oom:
+  BUS_SET_OOM (error);
+
+ failed:
+  _DBUS_ASSERT_ERROR_IS_SET (error);
+  if (reply)
+    dbus_message_unref (reply);
  out:
-  return retval;
+  return FALSE;
 }
 
 /* For speed it might be useful to sort this in order of
