diff -ur dbus-0.61.orig/qt/Makefile.am dbus-0.61/qt/Makefile.am
--- dbus-0.61.orig/qt/Makefile.am	2006-02-24 22:33:32.000000000 +0100
+++ dbus-0.61/qt/Makefile.am	2006-02-24 22:35:37.000000000 +0100
@@ -58,9 +58,9 @@
 	$(top_srcdir)/qt/qdbusxmlparser_p.h     
 
 
-$(top_srcdir)/qt/qdbusabstractadaptor.lo: qdbusabstractadaptor.moc
-$(top_srcdir)/qt/qdbusserver.lo: qdbusserver.moc
-$(top_srcdir)/qt/qdbusconnection.lo: qdbusconnection_p.moc
+qdbusabstractadaptor.lo: qdbusabstractadaptor.moc
+qdbusserver.lo: qdbusserver.moc
+qdbusconnection.lo: qdbusconnection_p.moc
 
 CLEANFILES=qdbusabstractadaptor.moc qdbusserver.moc qdbusconnection.moc
 
